<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminForm</class>
 <widget class="QDialog" name="AdminForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>修改教师信息</string>
    </attribute>
    <widget class="QTableView" name="infoTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>221</y>
       <width>311</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="relationTable">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>341</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>教师信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>执教关系</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="classText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>班级</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>工号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>科目</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subjectText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="FindButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sexBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>修改学生信息</string>
    </attribute>
    <widget class="QGroupBox" name="Box2_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>根据学生信息查找或添加</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="idText_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameText_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FindButton_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddButton_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Box2_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>根据教师查找</string>
     </property>
     <widget class="QLineEdit" name="teacherText_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Box2_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>80</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>根据班级查找</string>
     </property>
     <widget class="QLineEdit" name="classText_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="infoTable_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>651</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>高级修改模式</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string>user</string>
     </property>
     <item>
      <property name="text">
       <string>user</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>info</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>score</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>relation</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="FreshButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DoButtom">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>651</width>
       <height>341</height>
      </rect>
     </property>
     <property name="autoScrollMargin">
      <number>10</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="SqlOrder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>651</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>在此输入sqlite指令</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
